server/store/sqlstore/store.go

```diff
@@ -563,9 +563,15 @@ func (s *SQLStore) ListChatSubscriptionsToCheck() ([]storemodels.ChatSubscriptio
 	for rows.Next() {
 		var subscription storemodels.ChatSubscription
 		var expiresOn int64
-		if scanErr := rows.Scan(&subscription.SubscriptionID, &subscription.UserID, &subscription.Secret, &expiresOn, &subscription.Certificate); scanErr != nil {
+		var certificate *string
+		if scanErr := rows.Scan(&subscription.SubscriptionID, &subscription.UserID, &subscription.Secret, &expiresOn, &certificate); scanErr != nil {
 			return nil, scanErr
 		}
+		if certificate == nil {
+			subscription.Certificate = ""
+		} else {
+			subscription.Certificate = *certificate
+		}
```

Comment:
Can't we just do this?
```
if certificate != nil {
    subscription.Certificate = *certificate
}
```

Reply:
We should initialize the `subscription` variable such the `Certificate` field already has the value `""`.

Reply:
Good point

Reply:
Done